core/src/main/scala/cats/data/NonEmptyVector.scala

```diff
@@ -216,6 +216,9 @@ private[data] sealed trait NonEmptyVectorInstances {
       override def foldRight[A, B](fa: NonEmptyVector[A], lb: Eval[B])(f: (A, Eval[B]) => Eval[B]): Eval[B] =
         fa.foldRight(lb)(f)
 
+      override def get[A](fa: NonEmptyVector[A])(idx: Long): Option[A] =
+        fa.get(idx.toInt)
```

Comment:
I used a `Long` because the `size` is a `Long`. It seems strange that your collection can be a certain `size` without allowing you to get elements close to the end.